data/json/monster_factions.json

```diff
@@ -404,7 +409,8 @@
     "type": "MONSTER_FACTION",
     "name": "mi-go",
     "base_faction": "nether",
-    "neutral": [ "fish", "small_animal", "nether", "slime" ],
+    "neutral": [ "fish", "small_animal", "slime" ],
+    "by_mood": [ "nether" ],
```

Comment:
Reason for the change?

Reply:
They are canonically distinct from the generic grab-bag that are nether creatures, what with driving spaceships and such. Figured it's high time it shows up in-game. Slimes I'd make omnicidal as well, but per some discussion with Kevin I only get to do that if I gate their slimifying attack a bit better.